test/jdk/sun/java2d/marlin/ClipShapeTest.java

```diff
@@ -62,10 +62,43 @@
  * Note: Use the argument -slow to run more intensive tests (too much time)
  *
  * @run main/othervm/timeout=300 -Dsun.java2d.renderer=sun.java2d.marlin.DMarlinRenderingEngine ClipShapeTest -poly
+ */
+
+/*
+ * @test id=PolyDoDash
+ * @bug 8191814
+ * @summary Verifies that Marlin rendering generates the same
+ * images with and without clipping optimization with all possible
+ * stroke (cap/join) and/or dashes or fill modes (EO rules)
+ * for paths made of either 9 lines, 4 quads, 2 cubics (random)
+ * Note: Use the argument -slow to run more intensive tests (too much time)
+ *
```

Comment:
I think our style is only put these in the first `@run` block, and the rest to only carry the essentials for the tests to run. Simplifies the patch as well.

Reply:
Especially with such a long description that spans four lines.